Remove bitprim seeders

sickpig opened the pull request:
Both testnet and mainnet seeder are inactive

sickpig committed:
Remove bitprim seeders

Both testnet and mainnet seeder are inactive

gandrewstone commented:
seed.deadalnix.me doesn't look to be a true seeder that reports multiple and random nodes (and the IP it reports does not respond to 8333), so we should remove it as well:

dig seed.deadalnix.me

; <<>> DiG 9.11.3-1ubuntu1.11-Ubuntu <<>> seed.deadalnix.me
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 49437
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 65494
;; QUESTION SECTION:
;seed.deadalnix.me.		IN	A

;; ANSWER SECTION:
seed.deadalnix.me.	3557	IN	A	198.245.51.212

$ telnet seed.deadalnix.me 8333
Trying 198.245.51.212...
Trying 2607:5300:60:6d4::1...
telnet: Unable to connect to remote host: Network is unreachable

sickpig commented:
I see will do

sickpig committed:
Remove Amaury SÉCHET's seeders.

Both return the same IP address which is impossible to connect to
on port 8333.

Once/if the situation will be fixed we could put these seeders back.
